README.md

```diff
@@ -8,16 +8,30 @@ Module that provides an interface for a user to view and change product options,
 
 ## Installation / Setup
 
-1. Ensure that the repo is cloned locally for use. Also ensure that MongoDB is downloaded and started, as it is the database that this module uses.
+### Setup -- These steps only need to be completed the first time this module is to be used.
+
+1. Ensure that the repo is cloned locally for use. Also ensure that Cassandra DB is downloaded and started, as it is the database that this module uses.
 1. From the root of this repo, run `npm install` to install dependencies.
 1. Following the installation of all dependencies, run `npm run build` to generate the script bundle that the module will use.
-1. Lastly, while MongoDB is started and available, run `npm run postinstall` to seed the database.
+1. Run `npm run genCSV` to generate two CSV files -- `items.csv` and `reviews.csv`. These will be used to seed our database. Bear in mind that this command is generating at least 30,000,000 results, so it will have to run for a few minutes. It wouldn't be unreasonable to go get a drink or snack during this time. Also be aware that these files are not small! Have at least 4 GB of available hard drive space for them.
+1. You will have to do some manual work in Cassandra to complete the database setup. Don't be intimidated! You can copy and paste lines. First, in your command line, while Cassandra is running, enter `cqlsh` to interact with Cassandra, then when you see something like this:
+
+> Connected to Test Cluster at 127.0.0.1:9042.\
+> [cqlsh 5.0.1 | Cassandra 4.0 | CQL spec 3.4.5 | Native protocol v4]\
+> Use HELP for help.\
+> cqlsh>
+
+enter this command: `CREATE KEYSPACE mykea WITH REPLICATION = { 'class' :    'SimpleStrategy', 'replication_factor' : 1 };`
 
-> These setup steps are only necessary the first time a user attempts to use this module. Assuming these steps have been completed successfully at least once, the user can simply:
+6. While Cassandra is started and the `mykea` keyspace has been created, run `npm run cassieSetup` to create the tables required by the module.
+7. Once the Cassandra setup is complete and the CSVs are generated, within the `cqlsh` command line, enter this command: `COPY mykea.items (id, title, description, "originalPrice", "salePrice", colors, sizes, liked, "inStock") FROM 'DataGeneration/items.csv' WITH DELIMITER=',' AND HEADER=TRUE ;`. Expect this to take several minutes, as it is copying millions of lines into the database.
```

Comment:
Question, this is definitely not needed but I'm wondering, in my code for postgres I created something like:
let copyQueryOne =`COPY mykea.items (id, title, description, "originalPrice", "salePrice", colors, sizes, liked, "inStock") FROM 'DataGeneration/items.csv' WITH DELIMITER=',' AND HEADER=TRUE ;`

let copyQueryTwo = `COPY mykea.reviews (id, "easeOfAssembly", "valueForMoney", "productQuality", appearance, "worksAsExpected", "overallRating", "createdAt", "iRecommendThisProduct", header, body) FROM 'DataGeneration/reviews.csv' WITH DELIMITER=',' AND HEADER=TRUE ;`

and then in node I had a string run during one of my scripts. In your code I'm thinking a perfect place could be:
  .then(() => {
    return cassie.execute(createReviews);
  })
.then(() => {
    return cassie.execute(copyQueryOne);
  })
.then(() => {
    return cassie.execute(copyQueryTwo);
  })
  .then(() => {
    return cassie.shutdown();
  })

I haven't used Cassandra so I don't know if this would work but I was just thinking, maybe it would make it easier and reduce potential mistakes from the user? Just a suggestion though, I think the way you're doing it here is also a good way to do it.

Reply:
It's a good thought, but unfortunately `COPY x FROM y` is a cqlsh command, not a Cassandra command. It _has to_ be done in the terminal that way. Believe me, I tried to simplify it more.